src/html/template-landing.html

```diff
@@ -0,0 +1,246 @@
+      </div>
+    </section>
+    <section class="usa-section usa-section-dark usa-graphic_list">
+      <div class="usa-grid usa-graphic_list-item">
+        <div class="usa-width-one-half usa-media_block">
+          <img class="usa-media_block-img"  src="{{ site.baseurl }}/assets/img/circle-124.png" alt="Alt text">
+          <div class="usa-media_block-body">
+            <h3>Graphic headings can vary.</h3>
+            <p>Graphic headings can be used a few different ways, depending on what your landing page is for. Highlight your values, specific program areas, or results.</p>
+          </div>
+        </div>
+        <div class="usa-width-one-half usa-media_block">
+          <img class="usa-media_block-img"  src="{{ site.baseurl }}/assets/img/circle-124.png" alt="Alt text">
+          <div class="usa-media_block-body">
+            <h3>Stick to 6 or fewer words.</h3>
+            <p>Keep body text to about 30. They can be shorter, but try to be somewhat balanced across all four. It creates a clean appearance with good spacing.</p>
+          </div>
+        </div>
+      </div>
+      <div class="usa-grid usa-graphic_list-item">
+        <div class="usa-width-one-half usa-media_block">
+          <img class="usa-media_block-img"  src="{{ site.baseurl }}/assets/img/circle-124.png" alt="Alt text">
+          <div class="usa-media_block-body">
+            <h3>Never highlight anything without a goal.</h3>
+            <p>For anything you want to highlight here, understand what your users know now, and what activity or impression you want from them after they see it.</p>
+          </div>
+        </div>
+        <div class="usa-width-one-half usa-media_block">
+          <img class="usa-media_block-img"  src="{{ site.baseurl }}/assets/img/circle-124.png" alt="Alt text">
+          <div class="usa-media_block-body">
+            <h3>Could also have 2 or 6.</h3>
+            <p>In addition to your goal, find out your users’ goals. What do they want to know or do that supports your mission? Use these headings to show those.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="usa-section">
+      <div class="usa-grid">
+        <h2>Section heading</h2>
+        <p class="usa-font-lead">Everything up to this point should help people understand your agency or project: who you are, your goal or mission, and how you approach it. Use this section to encourage them to act. Describe why they should get in touch here, and use an active verb on the button below. “Get in touch,” “Learn more,” and so on.</p>
+        <a class="usa-button usa-button-big" href="#">Call to action</a>
+      </div>
+    </section>
+  </main>
+  <footer class="usa-footer usa-footer-medium" role="contentinfo">
+    <div class="usa-grid usa-footer-return-to-top">
+      <a href="#">Return to top</a>
+    </div>
+    <div class="usa-footer-primary-section">
```

Comment:
Yes, I like either of those, but I'm not sure if we can make that fix in this PR since it is a separate component, the footer, and this PR is more about the implementation of existing components. Instead, maybe we can open an issue around refactoring the footer so the next team can address it when they work on footers? What do you think?